python_scripts/numTours.py

```diff
@@ -0,0 +1,27 @@
+"""
+Run this at the end. Adds numTours variable to db.
```

Comment:
Can you talk about in the README about the fact that this script is used for the numTours variable used in the visualization?